cpp/src/datetime/datetime_ops.cu

```diff
@@ -25,9 +25,13 @@
 #include <cudf/table/table_view.hpp>
 #include <cudf/types.hpp>
 #include <cudf/utilities/traits.hpp>
+#include <cudf/wrappers/durations.hpp>
 
+#include <memory>
```

Comment:
`std` header should come lastly. And typically we don't need to include `<memory>` because it has been included somewhere else.